orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -271,12 +272,20 @@ class OperationsController {
       }
       trigger.buildInfo = buildInfo
       if (trigger.propertyFile) {
-        trigger.properties = buildService.getPropertyFile(
-          trigger.buildNumber as Integer,
-          trigger.propertyFile as String,
-          trigger.master as String,
-          trigger.job as String
-        )
+        try {
+          trigger.properties = buildService.getPropertyFile(
+            trigger.buildNumber as Integer,
+            trigger.propertyFile as String,
+            trigger.master as String,
+            trigger.job as String
+          )
+        } catch (RetrofitError e) {
+          if (e.response?.status == 404) {
+            throw new IllegalStateException("Expected properties file " + trigger.propertyFile + " (configured on Jenkins trigger), but it was missing")
```

Comment:
if `trigger.type` is available, that would be better than hard coding `Jenkins` here; LGTM otherwise

Reply:
If this is a manual trigger then the type is "manual" - so I don't want to use that value. Would it be better to just say "(configured on trigger)" @anotherchrisberry @erikmunson ?

Reply:
settled on just 'trigger'